app/upgrades/v25/upgrades.go

```diff
@@ -38,6 +40,33 @@ func CreateUpgradeHandler(
 		authenticatorParams.IsSmartAccountActive = false
 		keepers.SmartAccountKeeper.SetParams(ctx, authenticatorParams)
 
+		// Astroport uploaded a new contract and attempted to migrate their cw pools prior to the v24 upgrade.
+		// Their proposal to migrate the pool code IDs was only possible post v24 upgrade,
+		// due to this the pools are being manually upgraded as part of v25
+		poolIds := []uint64{1564, 1567, 1568, 1569, 1570, 1572, 1579, 1616, 1617, 1635, 1654}
```

Comment:
Lets define these in constants.go so we dont define it twice in the test and here.

Reply:
https://github.com/osmosis-labs/osmosis/pull/8130/commits/b9d50a0a2f49dde37d2f2b80ee0ce9f99a755a87